docs_user/modules/proc_completing-prerequisites-for-migrating-ceph-rgw.adoc

```diff
@@ -139,13 +140,13 @@ network_config:
   addresses:
   - ip_netmask: {{ external_ip }}/{{ external_cidr }}
   routes: {{ external_host_routes }}
-  members:
+  members: []
   - type: interface
     name: nic3
     primary: true
 ----
 
-.. Add the External Network to the `baremetal.yaml` file that is used by `metalsmith`:
+.. Add the External Network to the baremetal file, for example, `/home/stack/composable_roles/network/baremetal_deployment.yaml` that is used by `metalsmith`. Also 'network_config_update' should be enabled for network propogation to target nodes during `os-net-config`:
```

Comment:
I think we should ask to always pass `network_config_update` to the templates.
Do we want to call this out with a Note instead of adding this statement on L149?
Basically I'm proposing to keep the the title as it was, and put a Note saying "you must add `network_config_update: true` to make sure `os-net-config` is triggered.
@klgill might definitely have a better proposal.